archive_query_log/orm.py

```diff
@@ -437,6 +451,35 @@ class Index:
         }
 
 
+WarcDirectAnswerParserType = Literal[
+    "xpath",
+]
+
+
+class WarcDirectAnswerParser(BaseDocument):
+    provider: InnerProviderId | None = Object(InnerProviderId)
+    url_pattern_regex: str | None = Keyword()
+    priority: float | None = RankFeature(positive_score_impact=True)
+    parser_type: WarcDirectAnswerParserType = Keyword()
+    xpath: str | None = Keyword()
+    url_xpath: str | None = Keyword()
+    title_xpath: str | None = Keyword()
+    text_xpath: str | None = Keyword()
```

Comment:
Does a direct answer generally have a URL, a title, and a text?
I would have expected only a text and maybe a URL as a source to the answer.